packages/inter-protocol/src/auction/offerBook.js

```diff
@@ -93,30 +144,54 @@ export const prepareScaledBidBook = baggage =>
         const { records } = this.state;
         return [...records.entries(M.gte(toBidScalingComparator(bidScaling)))];
       },
+      publishOffer(record) {
+        const key = toScaledRateOfferKey(record.bidScaling, record.seqNum);
+
+        return E(getBidDataRecorder(key)).write(
+          harden({
+            bidScaling: record.bidScaling,
+            wanted: record.wanted,
+            exitAfterBuy: record.exitAfterBuy,
+            timestamp: record.timestamp,
+            balance: record.seat.getCurrentAllocation().Bid,
```

Comment:
the product spec calls this `give`. Should we

 - change it here
 - translate the field name on the client
 - ask product if `balanace` is a good name (my preference)

cc @turadg